app/scripts/config.js.example

```diff
@@ -11,12 +11,12 @@
         },
         bounds: {
             northEast: {
-                lat: 40.137992,
-                lng: -74.955763
+                lat: 39.8859538151,
+                lng: -75.2804974809
             },
             southWest: {
-                lat: 39.867004,
-                lng: -75.280266
+                lat: 40.0355776511,
+                lng: -75.0358172163
```

Comment:
This is a stickier wicket than it appears, since we use these bounds both for zooming the map and limiting the search.

Firstly, these got reversed, which for some reason the map doesn't care about but it seems to cause the address search to revert to be unconstrained.

Once I switch them, the default view with the tighter zoom and some organizations cut off definitely looks better, but since we use these bounds for the search, it means any organization not showing up on the initial view also won't show up in the search (which I confirmed by trying "9201 Germantown Ave", which is the Woodmere Art Museum.  It's near the top of Chestnut Hill and just out side the bounds).

It seems like Philly is just big enough that it doesn't all fit in the zoomlevel we would like to see by default.

One way to deal with this would be to create separate, more manual settings for the default view--i.e. just provide default center coordinates zoomlevel for  `DiscoverController` to use.  That seems cleaner than trying to use some sort of buffer/padding...

Reply:
Shoot. Good catch. Yeah, I'll just separate the configuration for these two things. More explicit than doing some weird buffer thing.

Reply:
Updated. Split the config for search and map, map now uses zoom+center for more strict control of the initial map state. You'll have to update your config again.